README.md

```diff
@@ -190,6 +202,7 @@ NOTE: only works for output
 - this tool is likley to require Elasticsearch version 1.0.0 or higher
 - elasticdump (and elasticsearch in general) will create indices if they don't exist upon import
 - when exporting from elasticsearch, you can have export an entire index (`--input="http://localhost:9200/index"`) or a type of object from that index (`--input="http://localhost:9200/index/type"`).  This requires ElasticSearch 1.2.0 or higher
+- If elasticsearch is in a sub-directory, index and type must be provided with a separate argument (`--input="http://localhost:9200 --input-index=index/type"`). Using `--input-index=/` will include all indices and types.
```

Comment:
You should show what a "standard" install looks like, and also a "custom" one, and how you would use this flag